src/node_report.cc

```diff
@@ -294,6 +295,10 @@ static void WriteNodeReport(Isolate* isolate,
         static_cast<bool>(uv_loop_alive(env->event_loop())));
     writer.json_keyvalue("address",
         ValueToHexString(reinterpret_cast<int64_t>(env->event_loop())));
+
+    // Report Event loop idle time
+    uint64_t idle_time = uv_metrics_idle_time(env->event_loop());
+    writer.json_keyvalue("loop_idle_time", 1.0 * idle_time / 1e6);
```

Comment:
For other times in the report we specify the unit as part of the key (e.g. `kernelCpuSeconds`), we should do the same here (and probably also switch to using seconds here for consistency)

Reply:
fixed